### DISCUSSION TITLE
Got this error message while trying to build Hello_world project

### DISCUSSION THREAD

**Discussion by robertblackwell** [ocaml/dune#15152] [Q&A]
The package hello_world does not have any user defined stanzas
attached to it. If this is intentional, add (allow_empty) to the package
definition in the dune-project file
I searched the documentation on dune stanzas and found nothing on (allow_empty)

Here is the `dune-project` file. I filled in none of this it was all create automatically by the build process
```
(lang dune 3.2)

(name hello_world)

(generate_opam_files true)

(source
 (github username/reponame))

(authors "Author Name")

(maintainers "Maintainer Name")

(license LICENSE)

(documentation https://url/to/documentation)
(package
 (name hello_world)
 (synopsis "A short synopsis")
 (description "A longer description")
 (depends ocaml dune)
 (tags
  (topics "to describe" your project)))

; See the complete stanza docs at https://dune.readthedocs.io/en/stable/dune-files.html#dune-project

```
Any suggestions ?

**Comment by koji-k** [comment]
A description `public_name` lacks in `bin/dune file`.
That is a cause maybe.

**Comment by bbatsov** [comment]
Is this a recent change? I started getting it myself and I can't understand why.

**Comment by bbatsov** [comment]
My simple solution was the just delete everything after the first line - which is quite appropriate for toy projects.
